<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: qradialgradient(spread:pad, cx:0.807667, cy:0.239, radius:0.5, fx:1, fy:0, stop:0 rgba(255, 255, 255, 255), stop:0.546875 rgba(210, 255, 232, 255))
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="exit_Btn">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>20</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:10px;
background-color:#74E3C3;
color:#fff;
}

QPushButton:hover{
background-color:#fff;
color:#74E3C3;
}
</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1121</width>
      <height>650</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1120</width>
      <height>650</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1920</width>
      <height>1070</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="mainpg">
     <widget class="QPushButton" name="checkStatistic_Btn">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>250</y>
        <width>1031</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:10px;
background-color:#74E3C3;
color:#fff;
}

QPushButton:hover{
background-color:#fff;
color:#74E3C3;
}</string>
      </property>
      <property name="text">
       <string>문제풀이 통계 확인</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="RTchatBtn">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>440</y>
        <width>201</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:10px;
background-color:#74E3C3;
color:#fff;
}

QPushButton:hover{
background-color:#fff;
color:#74E3C3;
}</string>
      </property>
      <property name="text">
       <string>실시간 상담</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="checkQnA_Btn">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>440</y>
        <width>191</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:10px;
background-color:#74E3C3;
color:#fff;
}

QPushButton:hover{
background-color:#fff;
color:#74E3C3;
}</string>
      </property>
      <property name="text">
       <string>QnA 확인</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTextBrowser" name="name_teacher">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>120</y>
        <width>331</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="lineWidth">
       <number>4</number>
      </property>
     </widget>
     <widget class="QLabel" name="welcome">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>124</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>선생님, 환영합니다!</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="standby_name">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>380</y>
        <width>281</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="lineWidth">
       <number>4</number>
      </property>
     </widget>
     <widget class="QLabel" name="standby">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>350</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>아래 학생 계정이 대기중입니다.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pg1_stastics">
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>401</width>
        <height>441</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="lineWidth">
       <number>4</number>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollcontent">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>379</width>
         <height>1500</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>1500</height>
        </size>
       </property>
       <widget class="QTableView" name="student_ans">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>385</width>
          <height>1501</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>700</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLineEdit" name="input_student">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
border: 0px;
}</string>
      </property>
      <property name="text">
       <string>학생명을 입력해 주세요.</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="student_selectBtn">
      <property name="geometry">
       <rect>
        <x>291</x>
        <y>0</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:0px;
background-color:#74E3C3;
color:#fff;
}

QPushButton:hover{
background-color:#fff;
color:#74E3C3;
}</string>
      </property>
      <property name="text">
       <string>선택</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTextBrowser" name="question">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>100</y>
        <width>701</width>
        <height>281</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="lineWidth">
       <number>4</number>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>390</y>
        <width>701</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="O_XLayout"/>
     </widget>
     <widget class="QPushButton" name="to_mainBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>530</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔스퀘어라운드</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:10px;
background-color:#74E3C3;
color:#fff;
}

QPushButton:hover{
background-color:#fff;
color:#74E3C3;
}
</string>
      </property>
      <property name="text">
       <string>◀메인으로</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="to_QnABtn">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>530</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔스퀘어라운드</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:10px;
background-color:#74E3C3;
color:#fff;
}

QPushButton:hover{
background-color:#fff;
color:#74E3C3;
}
</string>
      </property>
      <property name="text">
       <string>QnA 확인</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_Qnum">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>50</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
border: 0px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="Qnum_selectBtn">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>50</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:0px;
background-color:#74E3C3;
color:#fff;
}

QPushButton:hover{
background-color:#fff;
color:#74E3C3;
}</string>
      </property>
      <property name="text">
       <string>선택</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="qnum">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>56</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>검색할 문항</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pg2_QnA">
     <widget class="QPushButton" name="to_statisticBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>530</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔스퀘어라운드</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:10px;
background-color:#74E3C3;
color:#fff;
}

QPushButton:hover{
background-color:#fff;
color:#74E3C3;
}
</string>
      </property>
      <property name="text">
       <string>통계 확인</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="to_RTchatBtn">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>530</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔스퀘어라운드</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:10px;
background-color:#74E3C3;
color:#fff;
}

QPushButton:hover{
background-color:#fff;
color:#74E3C3;
}
</string>
      </property>
      <property name="text">
       <string>학생 상담채팅</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTableWidget" name="QnATable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>611</width>
        <height>471</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="lineWidth">
       <number>4</number>
      </property>
     </widget>
     <widget class="QTextBrowser" name="Question">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>80</y>
        <width>501</width>
        <height>351</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="lineWidth">
       <number>4</number>
      </property>
     </widget>
     <widget class="QTextBrowser" name="QstuName">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>20</y>
        <width>501</width>
        <height>51</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="lineWidth">
       <number>4</number>
      </property>
     </widget>
     <widget class="QTextEdit" name="Answer">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>510</y>
        <width>791</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="lineWidth">
       <number>4</number>
      </property>
     </widget>
     <widget class="QPushButton" name="QnA_loadBtn">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>440</y>
        <width>501</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔스퀘어라운드</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:1px;
background-color:#74E3C3;
color:#fff;
}

QPushButton:hover{
background-color:#fff;
color:#74E3C3;
}
</string>
      </property>
      <property name="text">
       <string>QnA 목록 불러오기</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pg3_RTchat">
     <widget class="QPushButton" name="msg_sendBtn">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>440</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:0px;
background-color:#74E3C3;
color:#fff;
}

QPushButton:hover{
background-color:#fff;
color:#74E3C3;
}</string>
      </property>
      <property name="text">
       <string>전송하기</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTextBrowser" name="chat_space">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>1061</width>
        <height>401</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>4</number>
      </property>
     </widget>
     <widget class="QTextEdit" name="msg_input">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>440</y>
        <width>851</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>4</number>
      </property>
     </widget>
     <widget class="QPushButton" name="to_mainBtn2">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>530</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔스퀘어라운드</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:10px;
background-color:#74E3C3;
color:#fff;
}

QPushButton:hover{
background-color:#fff;
color:#74E3C3;
}
</string>
      </property>
      <property name="text">
       <string>메인으로▶</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="to_QnABtn2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>530</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔스퀘어라운드</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:10px;
background-color:#74E3C3;
color:#fff;
}

QPushButton:hover{
background-color:#fff;
color:#74E3C3;
}
</string>
      </property>
      <property name="text">
       <string>QnA 확인</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
